circuit_knitting/cutting/cut_wire_to_move.py

```diff
@@ -13,16 +13,34 @@
 """Function to transform a :class:`.CutWire` instruction to a :class:`.Move` instruction."""
 from __future__ import annotations
 
+from typing import Callable
 from itertools import groupby
 import numpy as np
 
-from qiskit.circuit import Qubit, QuantumCircuit
+from qiskit.circuit import Qubit, QuantumCircuit, Operation
 from qiskit.circuit.exceptions import CircuitError
 from qiskit.quantum_info import PauliList
 from circuit_knitting.cutting.instructions.move import Move
+from circuit_knitting.cutting.qpd.instructions import TwoQubitQPDGate
 
 
-def transform_cuts_to_moves(circuit: QuantumCircuit, /) -> QuantumCircuit:
+def cut_wires(circuit: QuantumCircuit, /) -> QuantumCircuit:
+    """Transform all :class:`.CutWire` instructions in a circuit to :class:`.Move` instructions marked for cutting.
+
+    The returned circuit will have one newly allocated qubit for every :class:`.CutWire` instruction.
```

Comment:
Should we link to the Piveteau/Sutter LOCC wire cutting paper appendix here? THey do a nice explanation of wire cutting as a 2q identity cut / move operation

Reply:
Yes, I plan on making a milestone for tidying up the docs by using more links out to explanation (especially in cutting tutorials. they're very verbose.)

Reply:
I think you mean issue, rather than milestone? We can add such an issue to the 0.3.1 milestone.